### PR TITLE
Set travis mandatory to 1.0

### PR DESCRIPTION
**Pull request by matbesancon**

### CODE REVIEW

**Comment by simonbyrne** [comment]
Perhaps do Appveyor as well?

**Comment by simonbyrne** [comment]
can also delete the script block from .travis.yml (the default should work) and add in `using Pkg` to the `after_success` blocks.

**Comment by matbesancon** [comment]
Done!
Also REQUIRE julia version set to 1.0

File: .travis.yml [hunk 1]
```diff
@@ -13,9 +13,6 @@ notifications:
 branches:
   only:
   - master
-script:
-  - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
-  - julia --check-bounds=yes -e 'Pkg.clone(pwd()); Pkg.build("Distributions"); Pkg.test("Distributions"; coverage=true)'
 after_success:
-  - julia -e 'cd(Pkg.dir("Distributions")); Pkg.add("Coverage"); using Coverage; Coveralls.submit(process_folder())'
+  - julia -e 'using Pkg; cd(Pkg.dir("Distributions")); Pkg.add("Coverage"); using Coverage; Coveralls.submit(process_folder())'
   - julia -e 'Pkg.add("Documenter"); cd(Pkg.dir("Distributions")); include(joinpath("docs", "make.jl"))'
```
Comment (simonbyrne) [inline .travis.yml]: need `using Pkg` here as well

File: REQUIRE [hunk 2]
```diff
@@ -1,4 +1,4 @@
-julia 0.7.0-beta.85
+julia 1.0
```
Comment (simonbyrne) [inline REQUIRE]: unless you're using deprecated functionality, probably better to make the `julia 0.7` to make it easier on people upgrading packages.

Comment (matbesancon) [reply REQUIRE]: doesn't it set an upper bound too?

Comment (simonbyrne) [reply REQUIRE]: no, you use 2 numbers to specify an upper bound.

**Comment by simonbyrne** [comment]
Thanks!

**Pull request merged**